sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/converters/AnalyzeRequestConverter.java

```diff
@@ -35,15 +35,17 @@ public static AnalyzeTextOptions map(com.azure.search.documents.indexes.implemen
         }
 
         if (obj.getCharFilters() != null) {
-            List<CharFilterName> charFilters =
-                obj.getCharFilters().stream().map(CharFilterNameConverter::map).collect(Collectors.toList());
+            CharFilterName[] charFilters = obj.getCharFilters().stream()
+                .map(CharFilterNameConverter::map)
+                .toArray(CharFilterName[]::new);
             analyzeTextOptions.setCharFilters(charFilters);
         }
 
 
         if (obj.getTokenFilters() != null) {
-            List<TokenFilterName> tokenFilters =
-                obj.getTokenFilters().stream().map(TokenFilterNameConverter::map).collect(Collectors.toList());
+            TokenFilterName[] tokenFilters = obj.getTokenFilters().stream()
+                .map(TokenFilterNameConverter::map)
+                .toArray(TokenFilterName[]::new);
```

Comment:
These shouldn't have to change. Input-output models should have setter overloads for both varargs and List.

Reply:
Reverted